test/cli.test.ts

```diff
@@ -44,6 +43,38 @@ describe('CLI Tests', () => {
       throw new Error(stderr);
     }
   });
+  it(`Can specify dependency list`, async () => {
+    const GRAMMAR_FILE = path.join(ROOT_DIR, 'examples/minimal-with-dep.pegjs');
+    const outTsName = path.join(ROOT_DIR, 'output/minimal-with-dep.ts');
+    const barSource = path.join(ROOT_DIR, 'examples/bar.ts');
+    const barDest = path.join(ROOT_DIR, 'output/bar.ts');
+    // Copy the dependency to the output directory
+    await exec(`cp "${barSource}" "${barDest}"`);
```

Comment:
Unless there's something I'm not seeing, won't this mostly fail on Windows?

Reply:
Does `cp` not work on Windows? I used double quotes `"` specifically so that Windows wouldn't complain...

Reply:
I use bash on windows, that's why it worked for me. Otherwise, you need to use `copy`.

Reply:
You can use [copyfiles](https://github.com/calvinmetcalf/copyfiles#readme) to have it work mostly everywhere, or just rewrite the test to not need copying.